### PR TITLE
[Rule Tuning] BBR Rule Tuning 1 - Tighten Indexes Edition

### PR DESCRIPTION
**Pull request by w0rk3r**
## Issues

Related to #3551 

## Summary

Tighten the indexes of rules that I didn't touch the logic as part of this tuning round.

### CODE REVIEW

**Review by Aegrah — APPROVED** [review]

File: rules_building_block/defense_evasion_indirect_command_exec_pcalua_forfiles.toml [hunk 1]
```diff
@@ -13,7 +13,7 @@ Identifies indirect command execution via Program Compatibility Assistant (pcalu
 """
 from = "now-119m"
 interval = "60m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.process-*"]
```
Comment (Samirbous) [inline rules_building_block/defense_evasion_indirect_command_exec_pcalua_forfiles.toml:16]: this rule is compatilbe with sysmon and 4688, maybe not purpose of this PR but could be another one.

File: rules_building_block/defense_evasion_installutil_command_activity.toml [hunk 2]
```diff
@@ -15,7 +15,7 @@ a trusted Windows utility.
 """
 from = "now-119m"
 interval = "60m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.process-*"]
```
Comment (Samirbous) [inline rules_building_block/defense_evasion_installutil_command_activity.toml:18]:
```suggestion
index = ["logs-endpoint.events.process-*", "logs-system.security*", "winlogbeat-*", "logs-windows.*", "endgame-*"]
```

File: rules_building_block/defense_evasion_indirect_command_exec_pcalua_forfiles.toml (hunk 1 above)
Comment (Samirbous) [inline rules_building_block/defense_evasion_indirect_command_exec_pcalua_forfiles.toml:16]:
```suggestion
index = ["logs-endpoint.events.process-*", "logs-system.security*", "winlogbeat-*", "logs-windows.*", "endgame-*"]
```

File: rules_building_block/defense_evasion_msdt_suspicious_diagcab.toml [hunk 3]
```diff
@@ -17,7 +17,7 @@ references = [
 ]
 from = "now-119m"
 interval = "60m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.process-*"]
```
Comment (Samirbous) [inline rules_building_block/defense_evasion_msdt_suspicious_diagcab.toml:20]:
```suggestion
index = ["logs-endpoint.events.process-*", "logs-system.security*", "winlogbeat-*", "logs-windows.*", "endgame-*"]
```

File: rules_building_block/defense_evasion_suspicious_msiexec_execution.toml [hunk 4]
```diff
@@ -18,7 +18,7 @@ references = [
 ]
 from = "now-119m"
 interval = "60m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.process-*"]
```
Comment (Samirbous) [inline rules_building_block/defense_evasion_suspicious_msiexec_execution.toml:21]:
```suggestion
index = ["logs-endpoint.events.process-*", "logs-system.security*", "winlogbeat-*", "logs-windows.*", "endgame-*"]
```

File: rules_building_block/defense_evasion_unusual_process_path_wbem.toml [hunk 5]
```diff
@@ -13,7 +13,7 @@ Identifies unusual processes running from the WBEM path, uncommon outside WMI-re
 """
 from = "now-119m"
 interval = "60m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.process-*"]
```
Comment (Samirbous) [inline rules_building_block/defense_evasion_unusual_process_path_wbem.toml:16]:
```suggestion
index = ["logs-endpoint.events.process-*", "logs-system.security*", "winlogbeat-*", "logs-windows.*", "endgame-*"]
```

File: rules_building_block/discovery_generic_process_discovery.toml [hunk 6]
```diff
@@ -15,7 +15,7 @@ This rule identifies the execution of commands that can be used to enumerate run
 enumerate processes to identify installed applications and security solutions.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.process-*"]
```
Comment (Samirbous) [inline rules_building_block/discovery_generic_process_discovery.toml:18]:
```suggestion
index = ["logs-endpoint.events.process-*", "logs-system.security*", "winlogbeat-*", "logs-windows.*", "endgame-*"]
```

File: rules_building_block/discovery_security_software_wmic.toml [hunk 7]
```diff
@@ -14,7 +14,7 @@ Identifies the use of Windows Management Instrumentation Command (WMIC) to disco
 such as AntiVirus or Host Firewall details.
 """
 from = "now-9m"
-index = ["logs-endpoint.events.*", "winlogbeat-*", "logs-windows.*", "endgame-*"]
+index = ["logs-endpoint.events.process-*", "winlogbeat-*", "logs-windows.*", "endgame-*"]
```
Comment (Samirbous) [inline rules_building_block/discovery_security_software_wmic.toml:17]:
```suggestion
index = ["logs-endpoint.events.process-*", "winlogbeat-*", "logs-windows.*", "endgame-*", "logs-system.security*"]
```

File: rules_building_block/discovery_system_service_discovery.toml [hunk 8]
```diff
@@ -15,7 +15,7 @@ Detects the usage of commonly used system service discovery techniques, which at
 after compromising a system in order to gain a better understanding of the environment and/or escalate privileges. 
 """
 from = "now-9m"
-index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"]
+index = ["winlogbeat-*", "logs-endpoint.events.process-*", "logs-windows.*", "endgame-*"]
```
Comment (Samirbous) [inline rules_building_block/discovery_system_service_discovery.toml:18]:
```suggestion
index = ["winlogbeat-*", "logs-endpoint.events.process-*", "logs-windows.*", "endgame-*", "logs-system.security*"]
```

File: rules_building_block/discovery_system_time_discovery.toml [hunk 9]
```diff
@@ -15,7 +15,7 @@ Detects the usage of commonly used system time discovery techniques, which attac
 phase after compromising a system. 
 """
 from = "now-9m"
-index = ["winlogbeat-*", "logs-endpoint.events.*", "logs-windows.*", "endgame-*"]
+index = ["winlogbeat-*", "logs-endpoint.events.process-*", "logs-windows.*", "endgame-*"]
```
Comment (Samirbous) [inline rules_building_block/discovery_system_time_discovery.toml:18]:
```suggestion
index = ["winlogbeat-*", "logs-endpoint.events.process-*", "logs-windows.*", "endgame-*", "logs-system.security*"]
```

File: rules_building_block/lateral_movement_at.toml [hunk 10]
```diff
@@ -14,7 +14,7 @@ execution could be indicative of adversary lateral movement.
 """
 from = "now-119m"
 interval = "60m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.process-*"]
```
Comment (Samirbous) [inline rules_building_block/lateral_movement_at.toml:17]:
```suggestion
index = ["logs-endpoint.events.process-*", "winlogbeat-*", "logs-windows.*", "endgame-*", "logs-system.security*"]
```

File: rules_building_block/execution_wmi_wbemtest.toml [hunk 11]
```diff
@@ -14,7 +14,7 @@ local or remote endpoints.
 """
 from = "now-119m"
 interval = "60m"
-index = ["logs-endpoint.events.*"]
+index = ["logs-endpoint.events.process*"]
```
Comment (Samirbous) [inline rules_building_block/execution_wmi_wbemtest.toml:17]:
```suggestion
index = ["logs-endpoint.events.process*", "winlogbeat-*", "logs-endpoint.events.process-*", "logs-windows.*", "endgame-*", "logs-system.security*"]
```

**Review by Samirbous — APPROVED** [review]
added winlog related indexes where appropriate.

File: rules_building_block/defense_evasion_indirect_command_exec_pcalua_forfiles.toml (hunk 1 above)
Comment (w0rk3r) [reply rules_building_block/defense_evasion_indirect_command_exec_pcalua_forfiles.toml:16]: ++++

File: rules_building_block/defense_evasion_suspicious_msiexec_execution.toml (hunk 4 above)
Comment (w0rk3r) [reply rules_building_block/defense_evasion_suspicious_msiexec_execution.toml:21]: Edited this one, as `process.args_count` is available on Wineventlogs

**Pull request merged**